components/proxy/src/test/java/com/hotels/styx/routing/handlers/PathPrefixRouterTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+  Copyright (C) 2013-2020 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.routing.handlers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableMap;
+import com.hotels.styx.api.HttpInterceptor;
+import com.hotels.styx.api.LiveHttpRequest;
+import com.hotels.styx.routing.RoutingObject;
+import com.hotels.styx.routing.config.Builtins;
+import com.hotels.styx.routing.config.RoutingObjectFactory;
+import com.hotels.styx.routing.config.StyxObjectDefinition;
+import com.hotels.styx.routing.config.StyxObjectReference;
+import com.hotels.styx.server.NoServiceConfiguredException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Mono;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class PathPrefixRouterTest {
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    public Map<StyxObjectReference, RoutingObject> routingObjects = new HashMap<>();
+    public RoutingObjectFactory.Context Routingcontext;
```

Comment:
The variable name is capitalised: `RoutingContext`. Is this intentional?

Reply:
No. It is not. WIll fix.